InformationScripting/styles/item/VOperatorQueryNode/default

```diff
@@ -4,7 +4,7 @@
 		<symbol>|</symbol>
 	</pipeOp>
 	<substractOp prototypes="../Symbol/default">
-		<symbol>-</symbol>
+		<symbol>|-</symbol>
```

Comment:
You should remove the style loading code and the text belonging to the union operator below.